### DISCUSSION TITLE
Markdown-powered built-in to-do list

### DISCUSSION THREAD

**Discussion by NiceSapien** [Sketchware-Pro/Sketchware-Pro#470] [Ideas]
I guess it would be a good feature if there was a built-in to-do list. Because most of the times, when I open my project, I forget about opening to-do list in another app.

It would be more useful if it was markdown-powered. Or atleast, just simple text for now, if u r lazy.

**Comment by j-px** [comment]
I thought of implementing a Markdown library in Sketchware Pro for better changelogs, and with a To-Do list in projects, it'd actually be worth adding. Sounds like a great idea to me!

**Reply by NiceSapien** [reply]
You know I'm stupid and i have no patience. When will you add it?

**Reply by j-px** [reply]
When? No idea. If you want a feature implemented immediately/soon, contributions are welcome.
